src/workerd/api/actor-state.h

```diff
@@ -369,13 +369,23 @@ class DurableObjectState: public jsg::Object {
   // Gets an array of accepted WebSockets matching the given tag.
   // Disconnected WebSockets are automatically removed from the list.
 
+  void setWebSocketAutoresponse(kj::String request, kj::String response);
+  // Sets an object-wide websocket auto response message for a specific
+  // request string. All websockets belonging to the same object must
+  // reply to request with response and store the request recived timestamp.
```

Comment:
Sorry for being out of the loop here, but was this discussed somewhere? Kenton left a comment about this on the design doc, but there aren't any replies so I'm not sure if/where this was discussed.

Reply:
This was part of the original proposal and stated in the docs. Other than that there was no additional discussion about this specificaly, but no one seemed to opose to this. Do you think we should do this per-connection?

Reply:
I'm not opposed to setting it object-wide, but when I was looking at the code I noticed the strings were set on the `HibernationManager` rather than the websocket itself and was slightly surprised (mainly because it _feels_ like it could go per connection, or per tag, but to be clear I haven't thought about this at all).

I definitely think it's worth thinking about, shouldn't take more than 5 min or so at actors meeting today :wink:

Reply:
Verdict is we'll do it object-wide, thanks for discussing!